src/coreclr/tools/Common/JitInterface/AsyncMethodDesc.cs

```diff
@@ -0,0 +1,142 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Diagnostics;
+using Internal.TypeSystem;
+
+namespace Internal.JitInterface
+{
+    /// <summary>
+    /// Represents the async-callable (CORINFO_CALLCONV_ASYNCCALL) variant of a Task/ValueTask returning method.
+    /// Mirrors the structure and usage pattern of <see cref="UnboxingMethodDesc"/>.
```

Comment:
```suggestion
```
This is good to mention in PR description, but I do not think it is useful as a code comment.